File: RepoTransBench/target_projects/Java/C#/knightliao_pikaQ/public_tests/ColumnsPublicTest.cs
using Xunit;

namespace KnightliaoPikaQ.PublicTests
{
    public static class Columns
    {
        public const string NAME = "name";
        public const string CAMPAIGN_ID = "campaignId";
    }

    public class ColumnsPublicTest
    {
        [Fact]
        public void TestNameConstantNotNull()
        {
            Assert.NotNull(Columns.NAME);
            Assert.True(Columns.NAME.Length > 2);
        }

        [Fact]
        public void TestCampaignIdConstantHasId()
        {
            Assert.EndsWith("Id", Columns.CAMPAIGN_ID, System.StringComparison.OrdinalIgnoreCase);
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/knightliao_pikaQ/public_tests/columns_public_test.go
package public_tests

import "testing"

const (
	NAME       = "name"
	CAMPAIGN_ID = "campaignId"
)

func TestNameConstantNotNull(t *testing.T) {
	if NAME == "" {
		t.Error("NAME should not be empty or nil")
	}
	if len(NAME) <= 2 {
		t.Errorf("NAME should have length > 2, got %v", len(NAME))
	}
}

func TestCampaignIdConstantHasId(t *testing.T) {
	l := len(CAMPAIGN_ID)
	if l < 2 || CAMPAIGN_ID[l-2:] != "Id" {
		t.Errorf("CAMPAIGN_ID %q does not end with Id", CAMPAIGN_ID)
	}
}